ghost/core/core/frontend/helpers/content_api_path.js

```diff
@@ -0,0 +1,29 @@
+const {SafeString} = require('../services/handlebars');
+const logging = require('@tryghost/logging');
+const {urlUtils} = require('../services/proxy');
+
+module.exports = async function content_api_path(options) { // eslint-disable-line camelcase
+    let result;
+    const absoluteUrlRequested = getAbsoluteOption(options);
+
+    try {
+        let path = urlUtils.urlFor('api', {type: 'content'}, absoluteUrlRequested);
+        result = new SafeString(path);
+    } catch (error) {
+        logging.error(error);
+        result = '';
+    }
+
+    return result;
+};
+
+function getAbsoluteOption(options) {
+    const absoluteOption = options && options.hash && options.hash.absolute;
+    if (absoluteOption === undefined || absoluteOption === 'true' || absoluteOption === true || absoluteOption === null) {
+        return true;
+    } else {
+        return false;
+    }
+}
+
+module.exports.async = true;
```

Comment:
I don't think this needs to be an async helper! It will be much more performant without it!

Reply:
You're also missing a newline at the bottom of this file - I would recommend installing an [EditorConfig](https://editorconfig.org/) plugin for your editor